units/systemd-syscfg.service

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# This file is part of systemd.
+#
+# systemd is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+
+[Unit]
+Description=Merge System Configuration Images into /etc/
+Documentation=man:systemd-syscfg.service(8)
+
+ConditionCapability=CAP_SYS_ADMIN
+ConditionDirectoryNotEmpty=|/etc/syscfgs
```

Comment:
@poettering I think we don't want to store syscfgs in /etc right? Otherwise we get into nasty recursion issues, especially when they are plain directories, where it just won't work and overlayfs will reject it with -ELOOP

Reply:
that should be fine